caom2-tap-server/src/main/java/ca/nrc/cadc/tap/caom2/Util.java

```diff
@@ -233,4 +242,30 @@ static boolean isUploadedTable(Table t, List<Table> tabs) {
         }
         return upload;
     }
+    
+       
+    static Table findTableWithColumn(Column c, List<Table> tabs, TapSchema ts) {
+        log.debug("find: " + c.getColumnName());
+        Table ret = null;
+        for (SchemaDesc sd : ts.getSchemaDescs()) {
+            for (TableDesc td : sd.getTableDescs()) {
+                for (ColumnDesc cd : td.getColumnDescs()) {
+                    log.debug("find/check: " + cd.getColumnName() + " vs " + c.getColumnName());
+                    if (cd.getColumnName().equalsIgnoreCase(c.getColumnName())) {
+                        // found matching column: check if the table is in use
+                        for (Table t : tabs) {
+                            log.debug("find/verify: " + td.getTableName() + " vs " + t.getWholeTableName());
+                            if (td.getTableName().equalsIgnoreCase(t.getWholeTableName())) {
```

Comment:
Having just looked at the ADQL spec, I've noticed that identifiers escaped using double quotes are case sensitive.

Reply:
We currently do a poor job of handling quoted identifiers consistently and people getting the wrong capitalisation has been an issue in the past that we want to just "make it work". So there are already other places using equalsIgnoreCase...